src/callbacks_step/visualization.jl

```diff
@@ -166,7 +179,7 @@ function (visualization_callback::VisualizationCallback)(integrator)
 end
 
 """
-    show_plot(plot_data, variable_names;
+    show_plot(visualization_callback, plot_data, variable_names;
```

Comment:
There are changes to documented API, so this is breaking. We should either enable the previous behavior again or include this only in a breaking release.

Reply:
I guess nobody is using it actively, but it's documented API.